From 00d3b8ee5402215862b8cf55df4a81b2e94ebfed Mon Sep 17 00:00:00 2001
From: Marco Meyer-Conde <marco@tcu.ac.jp>
Date: Tue, 12 Nov 2024 09:37:36 +0900
Subject: [PATCH] cstdint

---
 appl_grid/serialise_base.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appl_grid/serialise_base.h b/appl_grid/serialise_base.h
index 5960b35..25fc276 100644
--- a/appl_grid/serialise_base.h
+++ b/appl_grid/serialise_base.h
@@ -19,7 +19,7 @@
 #include <iostream>
 
 #include <zlib.h>
-
+#include <cstdint>
 
 namespace SB { 
 
-- 
2.17.1

